neo/neo.csproj

```diff
@@ -37,17 +37,18 @@
     <PackageReference Include="Microsoft.AspNetCore.WebSockets" Version="2.0.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.0.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.0" />
-    <PackageReference Include="Neo.VM" Version="2.0.3" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net47' ">
     <Reference Include="System.Security" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net47'">
-    <PackageReference Include="Replicon.Cryptography.SCrypt">
-      <Version>1.1.6.13</Version>
+    <PackageReference Include="Replicon.Cryptography.SCrypt" Version="1.1.6.13">
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Before `neo-vm` was in `Nuget`, is this change an oversight?

Reply:
Yes, I should not have committed that :)